Classes2/Employee.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Classes2
+{
+    //2
+    internal class Employee
+    {
+        private readonly string _name, _surname;
+        private string _position;
+        private double _experience, _tax_charge, _salary;
```

Comment:
use camelCase (_taxCharge)

Reply:
If you don't have specific logic for setters and getters - get rid of these fields and use properties `{get; set;}`